test/python_tsp/test/ut_utils.py

```diff
@@ -24,11 +24,15 @@ def check_agreement(sub_test_num, total_sub_tests, ut_name, y_pure, y_wrapper, r
     else:
         ut_failed = False
 
-    if ut_failed:
-        print(f"FAILED SUB-TEST {sub_test_num} out of {total_sub_tests}:: {ut_name}, Max. Abs. Error: {max_absolute_error:.4e}, Max. Rel. Error: {max_relative_error:.4e}")
-        print("y_p: ", y_pure)
-        print("y_w: ", y_wrapper)
-    else:
-        print(f"PASSED SUB-TEST {sub_test_num} out of {total_sub_tests}:: Test Name: {ut_name}, Max. Abs. Error: {max_absolute_error:.4e}, Max. Rel. Error: {max_relative_error:.4e}")
-        # print('y_p: ', y_pure)
-        # print('y_w: ', y_wrapper)
+    with open("tsp.txt", "a") as file:
```

Comment:
We need to delete ".txt" file once done with unit testing to avoid appending to existing file